_rules/image-button-non-empty-accessible-name-59796f.md

```diff
@@ -108,6 +112,14 @@ The image button has an [accessible name][] through the `aria-labelledby` attrib
 <div id="id1">Search</div>
 ```
 
+#### Passed Example 5
+
+This image button has an empty `alt` attribute, but a non-empty `title` attribute. This gives the button an non-empty [accessible name][] through the `title` attribute.
+
+```html
+<input type="image" src="/test-assets/shared/search-icon.svg" title="Search" alt="" />
+```
+
```

Comment:
Since this does indeed cause significant accessibility support issues, I don't think we can include it as a passing example. It's perfectly understandable / correct for testers to fail such image buttons.

```suggestion
```

Reply:
My idea was to clearly define a way to test this scenario, showing a pass/fail example to better understand how to tackle it. It was a real case found in a website but the definition was ambiguous, without clearly explain if it was a pass or a fail.

If specs clearly define it as a pass, in my opinion we should reiterate our feedback, asking tech to support it as per specs, adding maybe a clear note in Accessibility Support section.